packages/ssr-web/app/components/common/payment-link/index.hbs

```diff
@@ -23,7 +23,7 @@
       href={{@deepLinkPaymentURL}}
       data-test-payment-link-deep-link
     >
-      {{or @cta "Pay Business"}}
+      {{or @cta "Pay with Card Wallet"}}
```

Comment:
Make the default similar to the payment CTA in https://github.com/cardstack/cardstack/pull/2772